src/views/Config/BasicConfig/Retainer.vue

```diff
@@ -170,33 +163,26 @@
 
 <script setup>
 import { getRetainer, updateRetainer } from '@/api/extension'
-import FormItemLabel from '@/components/FormItemLabel'
+import FormItemLabel from '@/components/FormItemLabel.vue'
 import InputWithUnit from '@/components/InputWithUnit.vue'
+import Oneof from '@/components/Oneof.vue'
 import useDataNotSaveConfirm, { useCheckDataChanged } from '@/hooks/useDataNotSaveConfirm'
 import useFormRules from '@/hooks/useFormRules'
 import useI18nTl from '@/hooks/useI18nTl'
 import { ElMessage } from 'element-plus'
-import { cloneDeep } from 'lodash'
 import { computed, nextTick, onMounted, reactive, ref, watch } from 'vue'
 import { useStore } from 'vuex'
 
 const { tl, t } = useI18nTl('Extension')
 const store = useStore()
 const { createRequiredRule } = useFormRules()
 
-const DISABLED_VALUE = 'disabled'
 const VALUE_FOR_NO_VALUE = '0s'
-const expiryTimeUnits = [
-  { value: 's', label: t('Base.second') },
-  { value: 'm', label: t('Base.minute') },
-  { value: 'h', label: t('Base.hour') },
-]
-const keysNeedTrans = ['msg_expiry_interval', 'msg_clear_interval']
 
 let retainerConfig = reactive({
   max_payload_size: '1MB',
-  msg_clear_interval: DISABLED_VALUE,
-  msg_expiry_interval: DISABLED_VALUE,
+  msg_clear_interval: VALUE_FOR_NO_VALUE,
```

Comment:
Please make this `VALUE_FOR_NO_VALUE` more clear.

Reply:
I don't seem to understand, change the variable name?